sycl/include/CL/sycl/detail/backend_traits_xrt.hpp

```diff
@@ -0,0 +1,85 @@
+//===------- backend_traits_xrt.hpp - Backend traits for XRT -----*-C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the specializations of the sycl::detail::interop,
+// sycl::detail::BackendInput and sycl::detail::BackendReturn specialization for
+// the XRT backend.
+//
+// the the supported conversions are:
```

Comment:
OK, let's be more factual :-)
```suggestion
// The supported conversions are:
```